src/style.css

```diff
@@ -0,0 +1,58 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 2px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+}
+
+.stars--1 :not(:nth-child(n + 2)),
+.stars--2 :not(:nth-child(n + 3)),
+.stars--3 :not(:nth-child(n + 4)),
+.stars--4 :not(:nth-child(n + 5)),
+.stars--5 :not(:nth-child(n + 6)) {
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
```

Comment:
you already have this style for stars__star

Reply:
No, code for
.stars__star {
background-image: url(images/star.svg);
background-repeat: no-repeat;
}
at the same time, code for
.stars--4 :not(:nth-child(n + 5)),
.stars--5 :not(:nth-child(n + 6)) {
background-image: url(images/star-active.svg);
background-repeat: no-repeat;
}

these selectors have different images active or not-active